### ISSUE TITLE
Background always black

### ISSUE THREAD

**Issue by sfinkens** [pytroll/pydecorate#8]
The background of a logo or text element is always black. Specifying the background color has no effect. 

```python
import numpy as np
from PIL import Image

from pydecorate import DecoratorAGG
import aggdraw


# Create sample image
im = Image.fromarray(np.random.randint(low=0, high=255, size=(480, 640), dtype=np.uint8))

# Add logo & text
deco = DecoratorAGG(image=im)
font = aggdraw.Font('black', '/usr/share/fonts/dejavu/DejaVuSansCondensed.ttf', size=48)
deco.add_logo('mylogo.png', bg='white')
deco.add_text('Test Label', font=font, bg='yellow', height=None)

im.save('decorated.png')
```
**Result**:

![decorated](https://user-images.githubusercontent.com/1991007/46673754-cac4bc00-cbda-11e8-9df0-06f6c9bc8f38.png)

**Expected**: White/yellow background for logo/text. 

**Environment**: pydecorate-0.1.0, aggdraw-1.3. It worked fine with aggdraw-1.1-20051010.

**Comment by djhoese** [comment]
What specific version of aggdraw 1.3? There have been a lot of fixes and the most recent version is 1.3.7.

**Comment by sfinkens** [comment]
Upgrading resolves the problem. Thanks!
